packages/logistration/src/LoginRedirect.test.jsx

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import { screen } from '@testing-library/react';
 import { getAuthenticatedUser } from '@edx/frontend-platform/auth';
 import { renderWithRouter } from '@edx/frontend-enterprise-utils';
+// eslint-disable-next-line import/no-unresolved
 import '@testing-library/jest-dom/extend-expect';
```

Comment:
[curious] why did this import need to disable the `import/no-unresolved` ESLint rule when `import '@testing-library/jest-dom/extend-expect';` is used in several other files, seemingly without the ESlint error? Why do we only have to ignore the error here? Or, conversely, can we avoid needing to disable this rule since it doesn't seem relevant for the other places where we're importing this same package in other test files.